### ISSUE TITLE
Error when setting up ZSH

### ISSUE THREAD

**Issue by AlexeyRaga** [haslersn/any-nix-shell#2]
This fails on mac
```
$ any-nix-shell --info-right | source /dev/stdin
ERROR: wrong usage
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
```

**Comment by gpevnev** [comment]
Use GNU sed and everything should work fine:
`nix-env -iA nixpkgs.gnused`

**Comment by haslersn** [comment]
> Use GNU sed and everything should work fine:
> `nix-env -iA nixpkgs.gnused`

Would this be fixed by replacing `-r` by `-E`?

**Comment by haslersn** [comment]
This issue is solved after #22 when you package with

```nix
{
  buildInputs = with pkgs; [ bash gnused ];
  installPhase = with pkgs; ''
    mkdir -p $out/bin
    cp -r bin $out
    wrapProgram $out/bin/any-nix-shell --prefix PATH : $out/bin:${lib.makeBinPath buildInputs}
  '';
}
```
